k8s/postgresql/deployer/deploy_with_tests.sh

```diff
@@ -23,12 +23,26 @@ overlay_test_schema.py \
   --dest "/data/schema.yaml"
 rm -f /data-test/schema.yaml
 
-/bin/expand_config.py
-export NAME="$(/bin/print_config.py --param '{"x-google-marketplace": {"type": "NAME"}}')"
-export NAMESPACE="$(/bin/print_config.py --param '{"x-google-marketplace": {"type": "NAMESPACE"}}')"
+NAME="$(/bin/print_config.py \
+    --xtype NAME \
+    --values_mode raw)"
+NAMESPACE="$(/bin/print_config.py \
+    --xtype NAMESPACE \
+    --values_mode raw)"
+export NAME
+export NAMESPACE
 
 echo "Deploying application \"$NAME\" in test mode"
 
+app_uid=$(kubectl get "applications/$NAME" \
+  --namespace="$NAMESPACE" \
+  --output=jsonpath='{.metadata.uid}')
+app_api_version=$(kubectl get "applications/$NAME" \
+  --namespace="$NAMESPACE" \
+  --output=jsonpath='{.apiVersion}')
+
+/bin/expand_config.py --values_mode raw --app_uid "$app_uid"
```

Comment:
Although this fixes it, I think this is not the ideal way of calling this script. Since it expands the raw values into final values, `--value_mode raw` shouldn't need to be specified--in fact it's just confusing!

The problem stems from the fact that the flag is defined in the library and this utility is not overriding it. We should move the flag to `print_config`. The library should only take in an explicit parameter instead, and `expand_config` pass that value as `raw`. Thoughts?

Reply:
Agreed - this fix (and this code, generally) is a hack.

Hopefully we'll get a chance to prioritize https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/issues/218, but until then each time change deployers we risk having to update these custom deployers, which is sad.

Tangentially, I think `print_config.py` is the merging of a bunch of different use cases (e.g. extract keys, extract values, `create_manifests.sh` for envsubst, `create_manifests.sh` for helm, etc), and each of those use cases should get a special purpose utility.